api/package.json

```diff
@@ -11,6 +11,7 @@
     "standard": "^17.1.2"
   },
   "dependencies": {
+    "@filecoin-stations/spark-piece-indexer-repository": "^1.0.0",
```

Comment:
When I run `npm i` after making this change, I am getting the following error:

```
❯ npm i
npm ERR! code E404
npm ERR! 404 Not Found - GET https://registry.npmjs.org/@filecoin-stations%2fspark-piece-indexer-repository - Not found
npm ERR! 404 
npm ERR! 404  '@filecoin-stations/spark-piece-indexer-repository@^1.0.0' is not in this registry.
npm ERR! 404 
npm ERR! 404 Note that you can also install from a
npm ERR! 404 tarball, folder, http url, or git url.

npm ERR! A complete log of this run can be found in: /Users/bajtos/.npm/_logs/2024-09-23T10_24_06_293Z-debug-0.log
```

@juliangruber can you spot the problem? I am following the very same setup as we have in https://github.com/filecoin-station/spark-stats, where this works.

Reply:
What I am trying to achieve: When we run `npm ci -w api`, I want npm to install dependencies for the "repository" workspace too, because "api" depends on "repository".

Reply:
Found it: a typo in the package name, notice `-stations` (plural).